nix/per-system-outputs.nix

```diff
@@ -0,0 +1,13 @@
+# This file is part of the IOGX template and is documented at the link below:
+# https://www.github.com/input-output-hk/iogx#35-nixper-system-outputsnix
+
+{ nix, projects, ... }:
+
+{
+  documents = nix.plutus.latex-docs;
+
+  packages.plutus-metatheory-site = nix.plutus.plutus-metatheory-site;
+
+  # Needed by .github/workflows/hlint.yml
+  hlint = projects.default.hsPkgs.hlint.components.exe.hlint;
```

Comment:
Although... `.github/.hlint.yml` is:
```
- ignore: { }
```

So it's no-op, always, right?

We could delete it, and the relative workflow file, and rely on `hlint.enable = true;` in `./nix/formatters.nix`.
With this, `hlint` will be run with each `git commit`, and also manually with `pre-commit run hlint` and also automatically run in CI along with all other formatters.

But because pre-commit only works on committed files, in order to lint existing .hs sources, you will have to run `pre-commit run hlint --all-files` and fix everything up (which you'd wanna do in a separate PR).
Thoughts?